gmplot/gmplot.py

```diff
@@ -397,9 +398,18 @@ def write_circle(self, f, lat, long, size, settings):
     def write_polyline(self, f, path, settings):
         clickable = False
         geodesic = True
+        icons = ''
+        _icons = settings.get('icons', {})
         strokeColor = settings.get('color') or settings.get('edge_color')
         strokeOpacity = settings.get('edge_alpha')
         strokeWeight = settings.get('edge_width')
+        for k in _icons:
+            # icon argument must not be in quotes
+            if k == 'icon':
+                icons += '%s: %s, ' % (k, _icons[k])
+            else:
+                icons += "%s: '%s', " % (k, _icons[k])
```

Comment:
Is this correct? If `icons` is the following...
```python
icons=[
      {
        'icon': 'symbolOne',
        'offset': '0%'
      }, {
        'icon': 'symbolTwo',
        'offset': '50%'
      }, {
        'icon': 'symbolThree',
        'offset': '100%'
      }
    ]
```

...then `k` will be the first dict object (not the first key of the first dict object).

It looks like you'll need two loops here, something like:
```python
for icon in _icons:
    # (add `{` to `icons` string)
    for property in icon:
        # (add formatted `property` to `icons` string)
    # (add `}` to `icons` string)
```

Don't forget to remove the braces (`{` and `}`) from `f.write('icons: [{%s}],\n' % icons)` below; we've added them in the above for-loop.